javascript/javascript1/week3/TripTime.js

```diff
@@ -0,0 +1,30 @@
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432
+};
+
+const travelTimeInMinutes = getTimeOfTrip(travelInformation);
+console.log("InMinutes:" + travelTimeInMinutes); 
+
+let hoursAndMinutes = timeConvert(travelTimeInMinutes);
+console.log(travelTimeInMinutes + " minutes = " + hoursAndMinutes);
+
+//Time = Distance ÷ Speed
+function getTimeOfTrip(travelInformation) {
```

Comment:
again, i would put function declarations first. But maybe that's your style. As long as they are grouped together, this can work.